src/content/changelog/4-65-0-2020-05-18.md

```diff
@@ -0,0 +1,21 @@
+---
+title: '4.65.0: 2020-05-18'
+date: 2020-05-18T04:00:00.000Z
+version: 4.65.0
+changelog:
+  - API
+---
+
+### Added
+
+- The `managed_issue` field was added to the Open Support Ticket endpoint ([POST /support/tickets](/api/v4/support-tickets/#post)).
+    - This field designates if the Support ticket is in reference to a [Managed Service](https://www.linode.com/products/managed/).
+
+### Fixed
+
+- A bug was fixed that was preventing initiation of scheduled, cross-data center migrations.
```

Comment:
```suggestion
- A bug was fixed that was preventing initiation of scheduled, cross-datacenter migrations.
```

Reply:
I'm seeing inconsistent usage between "data center" and "datacenter" in past changelog updates. Most international standards recommend "data center, and in the past we referred to this feature as "cross data center," so I recommend going with that.

Reply:
Ah, making it consistent would be nice, but I'm good with "cross data center" also. The cross-data part looked weird to me before.